### ISSUE TITLE
React instead of Blaze?

### ISSUE THREAD

**Issue by deenfoxx** [mizzao/meteor-autocomplete#139]
This package looks great. Any suggestions on how to get it to work with React instead of Blaze templates?

**Comment by denzils** [comment]
So this definitely doesn't work with React? Not even worth playing around to get it to work?

**Comment by StorytellerCZ** [comment]
The best solution would be to decouple the client from any front-end framework (hence it could be implemented in other front-ends) and have existing Blaze functionality move to another package.

**Comment by mizzao** [comment]
That's true, you could separate the publish/subscribe part from the Blaze part, but when I originally started this package it took a bunch of work to integrate it with the frontend :-)

**Comment by StorytellerCZ** [comment]
Well, one step at a time.
